docs/guides/high-availability/concepts-multi-site.adoc

```diff
@@ -54,23 +51,23 @@ Monitoring is necessary to detect degraded setups.
 | Less than one minute
 
 | {jdgserver_name} node
-| Multiple {jdgserver_name} instances run in each site. If one instance fails, it takes a few seconds for the other nodes to notice the change. Sessions are stored in at least two {jdgserver_name} nodes, so a single node failure does not lead to data loss.
+| Multiple {jdgserver_name} instances run in each site. If one instance fails, it takes a few seconds for the other nodes to notice the change. Entities are stored in at least two {jdgserver_name} nodes, so a single node failure does not lead to data loss.
```

Comment:
This isn't the case with the default MULTI_SITE feature as we advise configuring num_owners=1 for the session caches as we have persistent sessions enabled.

Reply:
Session caches are not used with current MULTI_SITE.

Reply:
Really!? I missed or forgot that development :sweat_smile:. Where is it we bypass those caches?

In that case the `deploy-infinispan-kubernetes-crossdc.adoc` guide needs to be updated as we're explicitly telling users to configure the various session caches with num_owners=1

Reply:
The sessions are now stored only in the database. The problem was the old remote store was asynchronously replicating changes to the other site and therefore causing stale data in the second site which is a much bigger problem for A/A deployments.

Reply:
Yes we should update it. I will have a look

Reply:
@ryanemerson I addressed the num_owners=1 in latest changes. Could you please have a look